05_stringer/joshcarp/IPAddr_test.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var stdout = os.Stdout
+
+var tests = []struct {
+	input    IPAddr
+	expected string
+}{
```

Comment:
Most people are testing (and handling) partial/empty IPAddr ( eg. IPAddr{127}, IPAddr{} )